app/presenters/finder_presenter.rb

```diff
@@ -278,6 +278,12 @@ def all_content_finder?
     self.content_item['content_id'] == 'dd395436-9b40-41f3-8157-740a453ac972'
   end
 
+  # FIXME: This should be removed once we have a way to determine
+  # whether to display metadata in the finder definition
+  def eu_exit_finder
```

Comment:
As per @aliuk2012's suggestion it would be nice to add a `?` to the end of the method name so that it follows Rails conventions.